src/components/ont/OntRunLibrariesList.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-  <div class="ont-run-libraries">
-    <b-list-group class="ont-run-libraries-list-group">
-      <OntTube v-for="library in libraries" v-bind:key="library.id" v-bind="library">
+
+   <div class="ont-run-libraries" v-on:drop="drop" v-on:dragover="allowDrop" v-on:dragleave="endDrop" v-bind:class="{hover: hover}">
+    <b-list-group class="ont-run-libraries-list-group" >
+      <OntTube v-for="library in libraries" v-bind:key="library.id" v-bind="library" v-bind:selected="isLibrarySelected(library)">
```

Comment:
Hopefully dont need to pass in selected prop, can loop through unselected libraries instead